tools/mo/openvino/tools/mo/middle/InsertLayoutPropagationTransposes.py

```diff
@@ -94,8 +94,8 @@ def find_and_replace_pattern(self, graph: Graph):
                 mark_output_as_in_correct_layout(permute_node, 0)
 
                 # keep the reinterp_shape_node in NHWC layout
-                mark_input_as_in_correct_layout(reinterp_shape_node, 0)
-                mark_input_as_in_correct_layout(reinterp_shape_node, 1)
+                for in_port_id, _ in reinterp_shape_node.in_ports().items():
```

Comment:
Should we update appropriate transformation test to cover these changes? Or they already covered by existed one?

Reply:
discussed offline, disided not implement new test

Reply:
discussed, decided not to add a new tests